airflow/dags/sentry_loader/sentry_loader.yml

```diff
@@ -0,0 +1,3 @@
+operator: operators.SentryToGCSOperator
+bucket: "{{ env_var('CALITP_BUCKET__SENTRY_EVENTS') }}"
+project_slug: "{{ env_var('CALITP__SENTRY_PROJECT_SLUG') }}"
```

Comment:
I think this should actually be hard-coded since it's part of the task definition; for example if we wanted to also sync the benefits sentry project, we'd have another task YAML with `project_slug: benefits`

Also I'm now realizing we haven't discussed Sentry environments but I think we can filter in the warehouse if needed.